batcher/aligned/src/main.rs

```diff
@@ -102,6 +110,37 @@ pub struct SubmitArgs {
     keystore_path: Option<PathBuf>,
 }
 
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct DepositToBatcherArgs {
+    #[arg(
+        name = "Batcher Eth Address",
+        long = "conn",
+        default_value = "0x7969c5eD335650692Bc04293B07F5BF2e7A673C0"
+    )]
+    batcher_eth_address: String,
+    #[arg(
+        name = "Path to local keystore",
+        long = "keystore_path",
```

Comment:
the other commands/flags seem to be kebab-case
```suggestion
        long = "keystore-path",
```

Reply:
It is the same convetion in the other commands, I agree tho